packages/react-components/react-tree/src/components/TreeItemAside/useTreeItemAsideStyles.styles.ts

```diff
@@ -0,0 +1,45 @@
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import type { TreeItemAsideSlots, TreeItemAsideState } from './TreeItemAside.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+
+export const treeItemAsideClassNames: SlotClassNames<TreeItemAsideSlots> = {
+  root: 'fui-TreeItemAside',
+};
+
+/**
+ * Styles for the action icon slot
+ */
+const useStyles = makeStyles({
+  base: {
+    display: 'flex',
+    marginLeft: 'auto',
+    ...shorthands.gridArea('aside'),
+  },
+  actions: {
+    position: 'relative',
```

Comment:
```suggestion
```

I removed this and couldn't see any issues when using another aside component too, did I miss any other use case?